models/repo.go

```diff
@@ -254,6 +255,13 @@ func (repo *Repository) ComposeMetas() map[string]string {
 			"user":   repo.MustOwner().Name,
 			"repo":   repo.Name,
 		}
+		switch repo.ExternalTrackerStyle {
+		case "alphanumeric":
```

Comment:
Should we use constantS like `ISSUE_NAME_STYLE_ALPHANUMERIC` and `ISSUE_NAME_STYLE_NUMERIC` to avoid spell it every time we use?